src/lib.rs

```diff
@@ -146,6 +114,102 @@ extern crate prost;
 mod otelapis;
 pub use otelapis::opentelemetry;
 
+/// This modules defines the OpenTelemetry gRPC service definitions introduced
+/// in v0.19 which make Otel gRPC responses fallible and transporting error context
+/// back to the requestor. In prior versions of Otel, the gRPC responses contained
+/// no error context and were always successful.
+///
+/// The change from infallible to fallible is a major breaking change forcing implementors
+/// to course correct their handlers to return errors. This module standardises this handling
+/// and provides a simple way to handle the error context in a uniform way as far as use in
+/// tremor is concerned.
+///
+pub mod common {
```

Comment:
can we pull this in a own file?

Reply:
Ja. I'll tweak that once `protoc` in actions is limping along 👍

Reply:
Donezo